src/JwtMethod.php

```diff
@@ -0,0 +1,135 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Auth\Jwt;
+
+use Jose\Component\Checker\ClaimCheckerManager;
+use Jose\Component\Checker\ExpirationTimeChecker;
+use Psr\Http\Message\ResponseInterface;
+use Psr\Http\Message\ServerRequestInterface;
+use Yiisoft\Auth\AuthenticationMethodInterface;
+use Yiisoft\Auth\IdentityInterface;
+use Yiisoft\Auth\IdentityRepositoryInterface;
+use Yiisoft\Http\Header;
+
+/**
+ * Authentication method based on JWT token.
+ */
+final class JwtMethod implements AuthenticationMethodInterface
```

Comment:
Not enough tests

![image](https://user-images.githubusercontent.com/1677515/94279596-00dce100-ff55-11ea-9b1a-e6e8a287e874.png)

Reply:
Increased test coverage